google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkPanel.java

```diff
@@ -43,12 +45,19 @@
 public class CloudSdkPanel {
 
   private TextFieldWithBrowseButton cloudSdkDirectoryField;
-  private JLabel warningMessage;
+  private JTextPane warningMessage;
   private JPanel cloudSdkPanel;
+  private JLabel warningIcon;
+
+  private static final String CLOUD_SDK_DOWNLOAD_LINK = "https://cloud.google.com/sdk/docs/"
```

Comment:
actually, since this is just a static (hardcoded) url that we verify - and not something generated by user input for example - are you ok with leaving it as a string? I don't really see the added benefit.